diff --git a/models/clubModel.js b/models/clubModel.js
index c667610..21c2f2e 100644
--- a/models/clubModel.js
+++ b/models/clubModel.js
@@ -5,7 +5,18 @@ const db = require('../database/db');
 exports.findClubById = async (clubId) => {
   const [rows] = await db.query(
     `
-    SELECT clubId, clubName, schoolId, categoryId, briefDescription, description, activity, profileImageUrl
+    SELECT 
+      clubId,
+      clubName,
+      schoolId,
+      categoryId,
+      briefDescription,
+      description,
+      activity,
+      recruitStartAt,
+      recruitEndAt,
+      profileImageUrl,
+      coverImageUrl
     FROM clubs
     WHERE clubId = ?
     `,
@@ -129,7 +140,7 @@ exports.getClubHistory = async (clubId, { page, pageSize }) => {
   const [rows] = await db.query(
     `SELECT 
       h.historyId,
-      u.userName AS modifier,
+      COALESCE(u.userName, '관리자') AS modifier,
       h.modifiedField,
       h.oldValue,
       h.newValue,
@@ -326,5 +337,30 @@ exports.createNewClub = async (clubData) => {
       clubData.lastModifiedBy
     ]
   );
+  return result.insertId;
+};
+
+exports.insertClubHistory = async ({
+  clubId,
+  userId,
+  modifiedField,
+  oldValue,
+  newValue,
+}) => {
+  const [result] = await db.query(
+    `
+    INSERT INTO histories (
+      clubId, userId, modifiedField, oldValue, newValue
+    ) VALUES (?, ?, ?, ?, ?)
+    `,
+    [
+      clubId,
+      userId,
+      modifiedField,
+      oldValue,
+      newValue,
+    ]
+  );
+
   return result.insertId;
 };
\ No newline at end of file
